server/build.gradle

```diff
@@ -32,15 +32,15 @@ dependencies {
   implementation 'io.javalin:javalin:3.12.0'
 
   // Mongo DB Driver for Java
-  implementation 'org.mongodb:mongodb-driver:3.12.7'
+  implementation 'org.mongodb:mongo-java-driver:3.12.7'
 
   // Jackson, a JSON library for Java
-  implementation 'com.fasterxml.jackson.core:jackson-databind:2.10.2'
-  // implementation 'com.fasterxml.jackson.core:jackson-databind:2.12.0'
+  // implementation 'com.fasterxml.jackson.core:jackson-databind:2.10.2'
+  implementation 'com.fasterxml.jackson.core:jackson-databind:2.12.0'
 
   // MongoJack, MongoDB integration for Jackson
-  implementation 'org.mongojack:mongojack:2.10.0'
-  // implementation 'org.mongojack:mongojack:4.0.2'
+  // implementation 'org.mongojack:mongojack:2.10.0'
+  implementation 'org.mongojack:mongojack:4.0.2'
```

Comment:
Coming from mongojack 2.x to 3.x and newer there is a migration guide to follow: https://github.com/mongojack/mongojack/blob/master/MIGRATING.md